nexus_constructor/model/group.py

```diff
@@ -82,6 +79,11 @@ def as_dict(self) -> Dict[str, Any]:
             if self.children
             else [],
         }
+        if self.attributes:
+            return_dict["attributes"] = [
+                attribute.as_dict() for attribute in self.attributes
+            ]
+        return return_dict
```

Comment:
I'm still seeing `"attributes": null,` in some places. My guess would be that the problem is that `self.attributes` can be true but the only attribute that is present is in the exclude list. Solution would be to make the list of attributes first and then test if it is empty.